test/EFCore.PG.FunctionalTests/BuiltInDataTypesNpgsqlTest.cs

```diff
@@ -108,7 +108,7 @@ public virtual void Can_query_using_any_mapped_data_type()
 
                         StringAsText = "Gumball Rules!",
                         StringAsVarchar = "Gumball Rules OK",
-                        CharAsChar1 = 'f',
+                        // CharAsChar1 = 'f',
```

Comment:
Could you add a `TODO` to this file, just to make sure we circle back?

Reply:
Sure, although I'm not sure that's what's going to make us remember :)